readability-research.md

```diff
@@ -0,0 +1,127 @@
+
+#### Advantages of CLEAR
+- Large - Much larger than any available corpora that provide readability criterion based on human judgement. Other large corpora (such as Newsela) do not provide readability criterion for individual texts, and corpora that do are much smaller (N = ~60-200 texts)
+- Broad - Curated from texts available through CommonLit, which selected excerpts from Wikipedia, Project Gutenberg, and other digital libraries. The text excerpts were also published over a very long period of time (1791 - 2020). 
+- Texts were hand-selected and hand-edited, ensuring high quality and minimal errors. 
+- The CLEAR corpus also includes metadata which provides opportunity for subanalyses. 
+
+#### Limitations of CLEAR
+- Readability criteria are based on judgements from schoolteachers about how difficult the excerpts would be for their students to read. 
+
+
+## Paper 2: # Perusal of readability with focus on web content understandability
+
+>[!NOTE]
+>Year: 2021
+>
+>Citations:  18
+
+*Pawan Kumar Ojha, Abid Ismail, Kuppusamy Kundumani Srinivasan,
+Perusal of readability with focus on web content understandability, Journal of King Saud University - Computer and Information Sciences, Volume 33, Issue 1, 2021, Pages 1-10, ISSN 1319-1578,
+https://doi.org/10.1016/j.jksuci.2018.03.007.*
+
+### Summary of abstract
+This paper:
+- Analyzes the popular readability indices used to check the readability of websites and web pages
+- Makes suggestions, based on survey findings, to consider while developing readability formulas
+
+### Conclusion / Findings
+- Most work on readability formulas still focuses on English only and will not work on texts in scripts other than used by English
+	- Most formulas estimate readability in terms of the US grading system 
+	- Formulas that focus on non-English languages include Djoko formula (Indonesian), Fernandez Huerta Index (Spanish), Kandal and Moles Index (French) and Al-Heeti (Arabic)
+- One limitation of formulas that mostly consider linguistic factors such as length and count of work is that this method could also score high on nonsense text
+- Readability methods developed using statistical language modeling may not be suitable for analyzing web content due to the presence of noise such as sidebar menus - when traditional formulas were used to analyze web content, they performed poorly
+- There is a lack of research discussing understandability of text for people with disabilities
+- Suggested measures to incorporate in improved readability formulas include: 
+	- structural components of the web such as hyperlinks and alt text
+	- weighting content - for example level 1 headings being weighted higher
+	- considering how to deal with dynamic content such as text within a collapsible element
+
+
+## Paper 3: # Towards Techniques for Easy-to-Read Web Content
```

Comment:
Interesting paper though it’s very specific to the Easy-to-Read use cases. Found this reference interesting: [(2012) W3C Easy-to-Read on the Web symposium](https://www.w3.org/WAI/RD/2012/easy-to-read/). And the proposed rules that are universal across languages:

- Write numbers as digits, not as words.
- Percentages and big numbers are hard to understand.
- Never use Roman numerals.
- Avoid all abbreviations like “e.g.”.
- Avoid using initials like “EU”.
- Avoid all special characters where possible.

And legibility rules:

- Always start a new sentence on a new line.
- Never use a hyphen to split a word over two lines.

---

Though those likely wouldn’t be suitable for inclusion directly in the CMS, it’s interesting to consider how we could devise a system that allows the creation of such rules for sites that have a need for this.